src/routes/safe/store/reducer/safe.ts

```diff
@@ -49,7 +50,7 @@ export default handleActions(
       const safeAddress = safe.address
 
       return state.updateIn(
-        [SAFE_REDUCER_ID, safeAddress],
+        ['safes', safeAddress],
```

Comment:
I think i got lost reviewing this but why did you removed the constant?

Reply:
the constant is the reducer's ID, while what we're looking here is the `safes` key inside the `safes` reducer. Unfortunately, we have the same name for the reducer and one of its attributes.